cmd/gossamer/config.go

```diff
@@ -326,6 +326,13 @@ func setDotRPCConfig(ctx *cli.Context, cfg *dot.RPCConfig) {
 // updateDotConfigFromGenesisJSON updates the configuration based on the genesis file values
 func updateDotConfigFromGenesisJSON(ctx *cli.Context, cfg *dot.Config) {
 
+	// use default genesis file if genesis configuration not provided, for example,
+	// if we load a toml configuration file without a defined genesis init value or
+	// if we pass an empty string as the genesis init value using the --geneis flag
+	if cfg.Init.Genesis == "" {
+		cfg.Init.Genesis = DefaultCfg.Init.Genesis
```

Comment:
I feel like this could cause bugs, is it possible to completely get rid of the need for a genesis file with the root command? that seems like it should be the expected behaviour

Reply:
expected behavior to have an empty genesis path in the toml configuration file?

Reply:
I can remove again but this has come up before. The only issue would be if someone uses a genesis file that has values they would prefer to the toml configuration values (the values that will be overwritten by the genesis values if a genesis path is provided) or if their config file has an empty genesis path value (which I think should be required and checked).

Reply:
I meant it's expected that you wouldn't need to provide a genesis file for the root command, when the node is already initialized. it would make more sense imo to not have it be required and checked if you're not initializing

Reply:
removed https://github.com/ChainSafe/gossamer/pull/795/files#diff-6e4a8b33cf573effe37b34dc984c597eL93